app/controllers/meetings_controller.rb

```diff
@@ -2,7 +2,12 @@ class MeetingsController < ApplicationController
   before_action :authenticate_user!
 
   def index
-    @meetings = current_user.meetings.all.order('start_time')
+    if current_user.admin?
+      meeting = Meeting.all
+    else
+      meeting = current_user.meetings
+    end
+    @meetings = meeting.all.order('start_time')
     @todays_meeting = @meetings.select {|x| x.start_time.day == Time.current.day}
     @upcoming_meeting = @meetings.select {|x| x.start_time.day > Time.current.day}
     @completed_meeting = @meetings.select {|x| x.start_time.day < Time.current.day}
```

Comment:
Time.current.day instead
DateTime.today may also work.

Reply:
DateTime.today is not working instead DateTime.current.day is working